### PR TITLE
Build CSV through boilerplate scripts

### PR DESCRIPTION
**Pull request by arjunrn**
This PR updates boilerplate with changes to build CSV through scripts in boilerplate. Also contains changes to files so that they are in the location expected by boilerplate scripts.

### CODE REVIEW

File: boilerplate/_data/last-boilerplate-commit [hunk 1]
```diff
@@ -1 +1 @@
-25bf1477435cc1d54ed1a0dff1c1f16405144db3
+ab74fa76e0db1413c653dc2a87c9cd541614dabf
```
Comment (2uasimojo) [inline boilerplate/_data/last-boilerplate-commit:1]: ✓

**Review by 2uasimojo — COMMENTED** [review]
/lgtm

from boilerplate.

**Comment by jharrington22** [comment]
/lgtm

OLM bundle contents look good.

**Pull request merged**